delete_student.py

```diff
@@ -0,0 +1,25 @@
+def random_st_delete(list_of_student):
+    for i in range(len(list_of_student)):
+        print(i+1, list_of_student[i])
+    print('are all students here? Answer y/n')
+    x = input()
+    if x == 'n':
+        print('who is absent today? print their №')
```

Comment:
We decided to extract all code that controls interaction with the user into the common part, so, please, find the appropriate functionality and reuse it.